configGlobal.go

```diff
@@ -96,15 +94,25 @@ func setGlobalConfigVariables(storageDir string, cfgFile string) error {
 	globalLockDir = storageDir
 	globalLogDir = filepath.Join(storageDir, "log")
 	globalLogFileCount = 5
+	onSuccessNotifiers = []Notifier{}
+	onFailureNotifiers = []Notifier{}
+	onStartNotifiers = []Notifier{}
 
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err != nil {
-		// No configuration file is okay unless we specifically asked for a named file
-		if cfgFile != "" {
-			fmt.Fprintln(os.Stdout, "Error:", err)
+		switch err.(type) {
+		default:
 			return err
+		case viper.ConfigFileNotFoundError:
+			// No configuration file is okay unless we specifically asked for a named file
+			if sendMail {
```

Comment:
Should this be: `if sendMail || testMailFlag {` ?

Reply:
If `testMailFlag` was set `duplicacy-util.go` would set `sendMail=true`. But I think your suggestion is easier to follow. I'll update this.